data/migrations/20220322032808_notes.js

```diff
@@ -0,0 +1,24 @@
+exports.up = (knex) => {
+  return knex.schema
+    .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
+    .createTable('notes', function (table) {
+      table
+        .uuid('note_id')
+        .unique()
+        .primary()
+        .defaultTo(knex.raw('uuid_generate_v4()'));
+      table.string('content_type').notNullable();
+      table.text('content').notNullable();
+      table.string('level').notNullable();
+      table.boolean('visible_to_admin').notNullable();
+      table.boolean('visible_to_moderator').notNullable();
+      table.boolean('visible_to_mentor').notNullable();
+      table.string('profile_id_mentor').notNullable();
```

Comment:
Lines 16 and 17 should be labeled "mentor_id" and "mentee_id" and should be foreign keys referencing the profiles table. Line 5 should have the primary key as notnullable. Let's set the primary key as increments instead of uuid to maintain consistencies across tables.